cfme/tests/automate/test_vmware_methods.py

```diff
@@ -29,6 +29,7 @@ def cls(domain):
     original_class = domain.parent\
         .instantiate(name='ManageIQ')\
         .namespaces.instantiate(name='System')\
+        .namespaces.instantiate(name='System')\
         .classes.instantiate(name='Request')
```

Comment:
Are you resolving any issue here?

Reply:
mistakenly added the extra line, now PR is updated please review